assignments/arrays.js

```diff
@@ -63,34 +63,66 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+
+let keys = Object.keys(inventory);
+let len = keys.length;
+// for (let i=0; i<len; i++) {
+//   console.log(inventory[i]);
+// }
+
+console.log('Car 33 is a ' + inventory[33].car_year.toString()
+            + ' ' + inventory[33].car_make.toString()
+            + ' ' + inventory[33].car_model.toString());
 
 
 
 // ==== Challenge 2 ====
 // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
-let lastCar = 0;
-console.log();
+let lastCar = inventory[len-1];
```

Comment:
Took me a second to realize where you pulled the ```len``` from. You may wanna define it directly above the first time you use it so that its a little more clear. Another way we could have done with without defining that variable is:
``` 
let lastCar = inventory[inventory.length-1];
console.log(lastCar);
```